webapp/store/actions/nodeActions.js

```diff
@@ -1,9 +1,9 @@
-import { bpanelClient } from '@bpanel/bpanel-utils';
+import { getClient } from '@bpanel/bpanel-utils';
 
 import * as types from '../constants/node';
 import { setChainInfo, getGenesisBlock } from './chainActions';
 
-const client = bpanelClient();
+const bpClient = getClient();
```

Comment:
any reason going from `client` to `bpClient`? I think `client` is a little more pleasant to the eye

Reply:
I guess trying to distinguish that this is our special "BPClient" rather than the singular clients. No other reason though. Happy to change it if it looks cleaner (I'm also getting self conscious about the fact that I'm basically leaving my initials everywhere).